cra-tf-validate-ignore-rules.json

```diff
@@ -0,0 +1,10 @@
+{
+  "scc_rules": [
+    {
+      "scc_rule_id": "rule-8cbd597c-7471-42bd-9c88-36b2696456e9",
```

Comment:
Where in the code are we creating a bucket? Id on't think we are, so why would CRA have to ignore this rule? We only create an instance (OCP creates the bucket but terraform plan has no notion of this)